shared/chat/conversation/index.desktop.js

```diff
@@ -107,6 +108,10 @@ class Conversation extends Component<void, Props, State> {
         </Text>
       </Box>
 
+    if (this.props.conversationIsError) {
```

Comment:
we shouldn't have these types of switching in the components. instead you should add a branch in the container